Download/Traducido-File.txt

```diff
@@ -0,0 +1,202 @@
+﻿
```

Comment:
Hello Mauricio this is an unnecesary file, for the PR you should remove it, please try to revove it before merging